CREDITS-LEVELS

```diff
@@ -125,3 +123,33 @@ Cluster 3
     DM30: Last Man Standing by Dean "deathz0r" Joseph
     DM31: Desolated Fort by Luiz Henrique "Xindage" Gasparin Jeronimo
     DM32: Fourplay by Brett "Mechadon" Harrell
+
+
+--Previously Featured in 0.12.1--
+
+E1M5: Main Control by Miguel "Z0k" Suarez Gomez
+E1M7: Transportation Bay by Will "Kid Airbag" Hackney (renamed, not removed)
+E1M9: Armory by Oleg "Wereknight" Vovk (moved to E1M5, not removed)
```

Comment:
This is a flat list where the secret levels (`E1M9` and `E2M9` in this case) are not indented under their respective levels. But how would you do that when the level in question is not mentioned because it was not changed since 0.12.1? As I previously mentioned my inclination would be to have a simple flat sorted list (like this), and this may be another reason to do so. There may not be anything for you to change here, just my thoughts on the topic.

Reply:
> But how would you do that when the level in question is not mentioned because it was not changed since 0.12.1?

Isn't that what the rest of the file is for?